src/i_sdlmusic.c

```diff
@@ -208,6 +210,12 @@ static boolean I_SDL_InitMusic(void)
         }
     }
 
+    #ifdef _WIN32
+    // Never let SDL Mixer use native midi on Windows. Avoids SDL Mixer bug
+    // where music volume affects global application volume.
+    putenv(sdl_mixer_disable_nativemidi);
```

Comment:
You could just pass the string directly to `putenv()` here. This will save you from two issues: First, you do not need to add a string literal to global context. And second, you won't forget to embrace that variable in `#ifdef _WIN32`. 😉

Reply:
Oh wow. Yes, "silliness" describes it best.

Anyway, please add the `#ifdef _WIN32` check around that variable then.

Reply:
I'll add a comment as well.